evap/results/templates/results_course_detail.html

```diff
@@ -34,12 +34,12 @@
         <div class="panel-heading">
             <span class="panel-title">{% trans "Overview" %}</span>
             {% if course.grades_activated and course.grade_documents.count == 1 and can_download_grades %}
-                <div class="pull-right btn-group">
+                <div class="pull-right btn-group hidden-print">
                     <a href="{% url "grades:download_grades" course.grade_documents.all.first.id %}" class="btn btn-xs btn-default" role="button" aria-expanded="false">{{ course.grade_documents.all.first.description }}</a>
                 </div>    
             {% elif course.grade_documents.count > 1 and can_download_grades %}
-                <div class="pull-right btn-group">
-                    <button type="button" class="btn btn-xs btn-default dropdown-toggle hidden-print" data-toggle="dropdown" aria-expanded="false">{% trans "Download grades" %} <span class="caret"></span></button>
+                <div class="pull-right btn-group hidden-print">
```

Comment:
please pull the div (and the closing tag) out of the if/elif to deduplicate this.